src/devices/__tests__/casparcg.spec.ts

```diff
@@ -187,6 +188,89 @@ describe('CasparCG', () => {
 			seek: 25 * 10
 		})
 	})
+	test('CasparCG: Play AMB for 60s in 50fps', async () => {
+
+		const commandReceiver0: any = jest.fn(() => {
+			return Promise.resolve()
+		})
+		let myLayerMapping0: MappingCasparCG = {
```

Comment:
Shouldn't all these be consts?

Reply:
Yeap, I've changed these in `casparcg.spec.ts` now but when we add better linting we'll have to do this everywhere